src/ocm/components/clusterWizard/StaticIp.tsx

```diff
@@ -60,7 +60,7 @@ const StaticIp: React.FC<StaticIpProps & { cluster: Cluster }> = ({
   return (
     <ClusterWizardStep navigation={<ClusterWizardNavigation cluster={cluster} />} footer={footer}>
       <WithErrorBoundary title="Failed to load Static IP step">
-        <StaticIpPage {...{ infraEnv, updateInfraEnv, onFormStateChange }} />
+        <StaticIpPage clusterId={cluster.id} {...{ infraEnv, updateInfraEnv, onFormStateChange }} />
```

Comment:
There a TS here now, ` clusterId` should not be passed.

Also please bring down the number of `max-warnings` in  `package.json` to 95